blocks/header/header.css

```diff
@@ -1153,6 +1153,11 @@ a.header__link {
     letter-spacing: 0.14px;
   }
 
+  .header__main-link-wrapper--tabs-with-cards .header__category-item p.with-inline-link,
```

Comment:
the class _with-inline-link_ will always have a _p_? or the inline-link class will always have a _a_? if this is the case, I'm ok with it